.github/workflows/cargo_hack.yml

```diff
@@ -1,8 +1,10 @@
 # A workflow that runs [cargo-hack](https://github.com/taiki-e/cargo-hack)
-# across the powerset of our features.
+# across our feature flags. We do not run a powerset as there's too many
+# features to fit all that in memory.
 name: cargo_hack
 
 on:
+  pull_request: {}
```

Comment:
Do we want this for PRs still?

Reply:
No, I'd like to cut this out once we have a sense of how long this run takes, especially if we can cut out `check-component`features` like you mentioned.